### PR TITLE
cardpeek 0.8.4

### PR DESCRIPTION
**Pull request by dunn**
The pixbuf stuff is to avoid this [GTK error where PNGs aren't recognized](http://drup.org/gtk-warning-error-loading-icon-couldnt-recognize-image-file-format).

### CODE REVIEW

**Comment by dunn** [comment]
also cc @DomT4 who might want to update his mirror

**Comment by DomT4** [comment]
Thanks for the nudge, Mirror updated!

**Comment by dunn** [comment]
Cool, PR updated.

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: cardpeek.rb [hunk 1]
```diff
@@ -24,22 +13,28 @@ class Cardpeek < Formula
...
+  depends_on "glib"
   depends_on "gtk+3"
   depends_on "lua"
+  depends_on "openssl"
+  depends_on :x11
 
   def install
-    # always run autoreconf, neeeded to generate configure for --HEAD,
+    ENV["GDK_PIXBUF_MODULEDIR"] = "#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
+    system "#{Formula["gdk-pixbuf"].opt_bin}/gdk-pixbuf-query-loaders", "--update-cache"
```
Comment (DomT4) [inline cardpeek.rb]: This may need to be shunted to `postinstall`, otherwise when pouring from bottle it won't get done on user's machines.

**Comment by dunn** [comment]
Rebuilding it now and that doesn't seem necessary at all anymore.

File: cardpeek.rb (hunk 1 above)
Comment (dunn) [inline cardpeek.rb]: Rebuilding it now and those lines don't seem necessary at all anymore.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-x11/commit/6eedd73b4ca81f7e65655f01634a1ccfb6f7e20e. Thanks @dunn!

**Pull request closed without merging**